zpi-backend/src/main/java/pwr/zpibackend/services/thesis/ThesisService.java

```diff
@@ -72,4 +85,62 @@ public Thesis updateThesis(Long id, Thesis param) {
         throw new NotFoundException();
     }
 
+    //  brakowało metody do usuwania tematu
+    //  co z rozłączaniem z employee/studentem itp? dobrze to jest?
+    @Transactional
```

Comment:
co do tej metody to nie wiem, ale zmerguj, ja przetestuje to dodając thesisDTO, wiem ze nie bylo, ja ją chciałem dopisać, przy następnym sprincie przy okazji tej możliwości usuwania starszych tematów od ustalonej daty